test/models/document.spec.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-unused-expressions */
+
+const expect = require('chai').expect;
+const model = require('../../app/models');
+const params = require('../testHelper.js');
+
+const documentParams = params.testDocument;
+const userParams = params.testUser;
+
+const requiredFields = ['title', 'content', 'OwnerId', 'access'];
+
+describe('Document Model', () => {
+  describe('How Document Model Works', () => {
+    let document;
+    let owner;
+
+    before(() => {
+      return model.Role.create(params.testRole)
+        .then((createdRole) => {
+          userParams.RoleId = createdRole.id;
+          return model.User.create(userParams);
+        })
+        .then((createdUser) => {
+          owner = createdUser;
+          documentParams.OwnerId = owner.id;
+        });
+    });
+
+    beforeEach(() => {
+      document = model.Document.build(documentParams);
+    });
+
+    after(() => {
+      return model.sequelize.sync({ force: true });
+    });
+
+    it('should be able to create a document', () => {
+      document.save()
+        .then((createdDocument) => {
+          expect(createdDocument).to.exist;
+          expect(typeof createdDocument).to.equal('object');
+        });
+    });
+    it('should create a document with title and content', () => {
```

Comment:
Looks like all the assertions in other spec below can be combined into one spec or use the requiredFields to loop through
Also, maybe add test for delete and update action
For delete actions that cascade to other models, you might want to check that too